docs/operate/customize/collections.md

```diff
@@ -0,0 +1,179 @@
+_Introduced in v13_
+
+Collections provide a powerful way to curate and organize records within your InvenioRDM instance. They define sets of records based on search filters and can be organized hierarchically to create meaningful groupings of content.
+
+## Overview
+
+Collections are dynamic, query-based groupings of records that automatically include all records matching a specified search filter. They enable you to:
+
+- **Organize records thematically** - Group records by subject, resource type, funding program, or any metadata field
+- **Create hierarchical structures** - Build nested collections to guide users in browsing records
+- **Display curated content** - Present collections on dedicated pages within communities or globally
+
+Each collection is stored in the database with a search query string that dynamically fetches matching records. This approach ensures collections stay current as new records are added to your repository.
+
+!!! info "Architecture Details"
+    For technical implementation details, see [RFC0079](https://github.com/inveniosoftware/rfcs/blob/master/rfcs/rdm-0079-collections.md).
```

Comment:
it would be nice for this information to be in Maintain and Develop rather than an RFC...